actions/workflows/get_error_data.yaml

```diff
@@ -0,0 +1,84 @@
+---
+version: 1.0
+
+description: Gets the execution tree and the resulting error of a given execution
+
+input:
+  - cmdb_request_item
+  - cmdb_request_item_url
+  - retire
```

Comment:
retirement is something that is encore specific. If we need to do certain things in certain situations we should do it on the encore side. This workflow should be user generic and just run both tasks and output the data for the user to use however they need.